src/main/java/study/step3/Car.java

```diff
@@ -1,4 +1,23 @@
 package study.step3;
-public interface Car{
-    void move(boolean condition);
+
+public class Car{
+    private static final Integer START_POSITION = 1;
+    private String carName;
+    private int position;
+    public Car(String carName){
+        this.carName = carName;
+        this.position = START_POSITION;
+    }
+    public void move(MoveStrategy moveStrategy){
+        if(moveStrategy.move() >= NumberChecker.CONDITION_OF_START){
+            position++;
+        }
```

Comment:
```suggestion
        if(moveStrategy.isMove()){
            position++;
        }
```
이런식으로 작동할 수 있게 만들어보면 어떨까요?